src/support/import-supervisor.js

```diff
@@ -213,6 +213,23 @@ function ImportSupervisor(services, config) {
     return newImportJob;
   }
 
+  /**
+   * Gets the Api Key metadata based on the hashed key.
+   * @param hashedKey
+   * @return {Promise<null|{name: *, imsUserId: *, imsOrgId: *}>}
+   */
+  async function getApiKeyMetadata(hashedApiKey) {
+    const metadata = await dataAccess.getApiKeyByHashedKey(hashedApiKey);
+    if (metadata) {
+      return {
+        imsOrgId: metadata.imsOrgId,
+        name: metadata.name,
```

Comment:
Just to be specific:

```suggestion
        apiKeyName: metadata.name,
```

Reply:
Are there accessor methods we can use on the API key model instead of direct property access? `getImsOrgId()`, `getName()` etc.